charts/karma/templates/deployment.yaml

```diff
@@ -80,6 +80,20 @@ spec:
               mountPath: /etc/certs/{{ . }}
               readOnly: true
           {{- end }}
+        {{- if .Values.kthxbyeSidecar.enabled }}
+        - name: kthxbye-sidecar
+          image: "{{ .Values.kthxbyeSidecar.image.repository }}:{{ .Values.kthxbyeSidecar.image.tag }}"
+          imagePullPolicy: "{{ .Values.kthxbyeSidecar.image.pullPolicy }}"
+          args:
+            - --listen=:80
```

Comment:
Should we listen to another port, non-restricted (> 1024) and not run as root?

Reply:
yes, I agree. I have replaced the port to 8082. also, have added default nonroot user in the kthxbye sidecar container when securityContext is enabled on the pod level.